src/com/dotcms/rest/api/v1/authentication/TokenUnvalidException.java

```diff
@@ -0,0 +1,27 @@
+package com.dotcms.rest.api.v1.authentication;
+
+/**
+ * Thrown when a Token is unvalid
+ */
+public class TokenUnvalidException extends Exception{
```

Comment:
It might be a good idea to use the naming convention that we have in place for our exceptions. This would be called "DotInvalidTokenException"